### DISCUSSION TITLE
What is Anamorphic option?

### DISCUSSION THREAD

**Discussion by voska7** [RandomEngy/VidCoder#1013] [Q&A]
![image](https://user-images.githubusercontent.com/42012426/166130406-3e3e4376-fc2f-4cda-8cb0-3e5ff6f92916.png)

what is anamorphics option? should i disable it?

**Comment by RandomEngy** [comment]
Enabling anamorphic allows the output video to have non-square pixels.

Advantages:
* Allows transcoding anamorphic input (DVDs) without resampling - you get better quality and smaller file size

Disadvantages:
* Not all players support this

Most players do support Anamorphic so you could leave it on unless you see issues with a squashed picture when you play them.
